include/odp/api/spec/crypto.h

```diff
@@ -403,6 +406,9 @@ typedef struct odp_crypto_op_param_t {
 	/** Override session IV pointer */
 	uint8_t *override_iv_ptr;
 
+	/** Override session authentication IV pointer */
+	uint8_t *override_auth_iv_ptr;
```

Comment:
The old override_iv_ptr is now ptr for cipher. In minimum, the documentation should say that. Although, while changing IV pointers, we could update these also.
/* Override IV pointer for cipher */
union {
DEPRECATED(override_iv_ptr);
cipher_iv_ptr;
}

/* Override IV pointer for auth */
auth_iv_ptr;

Reply:
@lumag @psavol Looks like we're very close here. Can this PR be finalized for Monday's ARCH call?

Reply:
@psavol @Bill-Fischofer-Linaro updated.